vendor/brave-ios/Ads/BATBraveAds.h

```diff
@@ -5,12 +5,12 @@
 #import <Foundation/Foundation.h>
 #import <UserNotifications/UserNotifications.h>
 
-typedef NS_ENUM(NSInteger, BATAdsNotificationEventType) {
-  BATAdsNotificationEventTypeViewed,     // = ads::NotificationEventType::VIEWED
-  BATAdsNotificationEventTypeClicked,    // = ads::NotificationEventType::CLICKED
-  BATAdsNotificationEventTypeDismissed,  // = ads::NotificationEventType::DISMISSED
-  BATAdsNotificationEventTypeTimedOut    // = ads::NotificationEventType::TIMEOUT
-} NS_SWIFT_NAME(NotificationEventType);
+typedef NS_ENUM(NSInteger, BATAdsAdNotificationEventType) {
```

Comment:
Ah, didn't realize this was renamed. Actually this will fail to build now (report methods need changed type name)

Reply:
Also im not sure I understand why it was renamed at all? Ads ad notification event type?

Reply:
@kylehickinson Was renamed as we now have AdNotifications and will shortly have PublisherAds

Reply:
Will these new publisher ads have a different enum with different values? I think `BATAdsNotificationEventType` and `BATPublisherAdsNotificationEventType` will be fine if they have to remain separate. If they're the same event regardless of if its a publisher ad or a regular ad then we can just keep `BATAdsNotificationEventType` alone

Reply:
More for consistency with `bat-native-ads` and when searching for AdNotification and PublisherAd that all occurrences appear

Reply:
Fixed build errors

Reply:
If we're going to have two enums then, I would rename it to `BATAdNotificationEventType` and `BATPublisherAdNotificationEventType` instead of `AdsAd`

Reply:
Changed to `BATAdNotificationEventType` and will use `BATPublisherAdEventType` for Publisher Ads